HIP/hip-865.md

```diff
@@ -0,0 +1,104 @@
+---
+hip: 865
+title: Add EVM Support for transient storage and memory copy Cancun opcodes
+author: Danno Ferrin (@shemnon)
+working-group: Nana Essilfie-Conduah <@nana-ec>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: 2024-01-24
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/872
+---
+
+## Abstract
+
+Update the Hedera network to add support for new Opcodes found in the Cancun fork of
+Ethereum Mainnet, namely TSTORE, TLOAD, and MCOPY
```

Comment:
In the _Motivation_ section you mentioned that this HIP excludes blob related features, should we mention it briefly here? Otherwise, by reading only this it seems that Cancun only includes these 3 new opcodes.

Reply:
FYI the _title_ of the HIP covers this (but your suggestion is still valid).

Reply:
Done.